x/auth/vesting/cmd/vestcalc/vestcalc.go

```diff
@@ -13,6 +13,9 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/auth/vesting/client/cli"
 )
 
+// vestcalc is a utility for creating or reading period files
```

Comment:
Vocabulary consistency nit.
```suggestion
// vestcalc is a utility for creating or reading schedule files
```

Reply:
Done.